src/plugins/remote-loader/index.ts

```diff
@@ -42,27 +44,28 @@ export async function remoteLoader(
   settings: LegacySettings
 ): Promise<Plugin[]> {
   const allPlugins: Plugin[] = []
-
   const pluginPromises = (settings.remotePlugins ?? []).map(
     async (remotePlugin) => {
-      try {
-        await loadScript(remotePlugin.url)
+      if (settings.integrations[remotePlugin.name]) {
```

Comment:
This creates an implicit dependency between `settings.integrations` and `remotePlugins`, which should not be the case. We can't guarantee that remote plugins have an integrations object counterpart because remote plugins are not exclusively destinations